### ISSUE TITLE
WCAG 3.1.1: <html> element must have a lang attribute (html)

### ISSUE THREAD

**Issue by AdnoC** [AdnoC/Dotfiles#3]
**Issue**: `<html> element must have a lang attribute` (`html-has-lang`: https://dequeuniversity.com/rules/axe/3.1/html-has-lang?application=axeAPI)

**Target application title**: xkcd: .NORM Normal File Format
**Target application url**: https://xkcd.com/

**Element path**: html

**Snippet**:

    <html>

**How to fix**:
Fix any of the following:
  The <html> element does not have a lang attribute

**Environment**:
Chrome version 71.0.3578.98

====

This accessibility issue was found using Accessibility Insights for Web, a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
